src/streaming.jl

```diff
@@ -215,7 +215,7 @@ function parsefile(filename::AbstractString,callbacks::XPCallbacks; bufferlines=
                 write(io, readline(file))
                 i += 1
             end
-            txt = Compat.bytestring(io)
+            txt = Compat.String(io)
```

Comment:
`@compat Compat.String` might work - it needs to be a syntax rewrite on 0.3 because that doesn't support call overloading, but it needs to be `Compat.String` on 0.4 because it was deprecated there. Alternately, drop 0.3 support by raising the minimum version in REQUIRE, remove 0.3 testing from CI, and when it gets tagged, bump the minor version number with `Pkg.tag("LibExpat", :minor)`.